server/webhook_jira.go

```diff
@@ -29,6 +31,50 @@ type JiraWebhook struct {
 	IssueEventTypeName string `json:"issue_event_type_name"`
 }
 
+func (jwh *JiraWebhook) expandIssue(p *Plugin, instanceID types.ID) error {
+	instance, err := p.instanceStore.LoadInstance(instanceID)
+	if err != nil {
+		return err
+	}
+
+	// Jira Cloud comment event. We need to fetch issue data because it is not expanded in webhook payload.
+	isCommentEvent := jwh.WebhookEvent == commentCreated || jwh.WebhookEvent == commentUpdated || jwh.WebhookEvent == commentDeleted
+	if isCommentEvent && instance.Common().IsCloudInstance() {
```

Comment:
I think we should check if it's a cloud instance first and return if Jira server, to show that this function doesn't care about Jira server first and foremost